client/index.js

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';
 import { Provider } from 'react-redux';
 import { Router } from 'react-router-dom';
 import history from './history';
 import store from './store';
 import App from './App';
 import './style.css';
 
-ReactDOM.render(
+const container = document.getElementById('app');
+const root = createRoot(container);
+root.render(

```

Comment:
This update removes a warning that keeps popping up with the new react version in console.

Let's remove the noise and be able to use console for debugging.